From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 22 May 2014 16:22:48 +0900
Subject: man: update URL refernce in daemon(7)

http://lists.freedesktop.org/archives/systemd-devel/2014-May/019410.html
(cherry picked from commit 0afedd300cbe53bd5a54aefc81cede1eae9a9ab8)
(cherry picked from commit 13fc6c9bfacdca10d3e7eb73cd96515cae760734)
---
 man/daemon.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/daemon.xml b/man/daemon.xml
index 170949e..543468e 100644
--- a/man/daemon.xml
+++ b/man/daemon.xml
@@ -352,7 +352,7 @@
 
                         <para>These recommendations are similar but
                         not identical to the <ulink
-                        url="http://developer.apple.com/mac/library/documentation/MacOSX/Conceptual/BPSystemStartup/Articles/LaunchOnDemandDaemons.html#//apple_ref/doc/uid/TP40001762-104738">Apple
+                        url="https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html">Apple
                         MacOS X Daemon Requirements</ulink>.</para>
                 </refsect2>
 
